roles/tekton-pipeline/templates/resources.yml.j2

```diff
@@ -43,10 +43,10 @@
     namespace: {{ username }}-cicd
   spec:
     accessModes:
-      - ReadWriteMany
+      - ReadWriteOnce
```

Comment:
1. They weren't being retained after task run, so they were effectively not functioning as a cache
2. A per-user cache will offer some benefit
3. RWX storage as a requirement makes this
  - more expensive (OCS) or more complex (NFS out of tree provisioner)
  - harder to deploy flexibly anywhere

So Andy and I talked about it and decided that the muted benefits of a per-user cache (the pipelines won't do more than one PipelineRun at a time, that's why it worked) outweigh the problems with requiring RWX (each one was separate anyways because the PVCs weren't in a shared namespace)